api-gateway/src/main/java/com/petclinic/bffapigateway/presentationlayer/v2/VisitController.java

```diff
@@ -128,7 +128,20 @@ public Flux<EmergencyResponseDTO> getAllEmergency(){
         return visitsServiceClient.getAllEmergency();
     }
 
-    @GetMapping(value="/emergency/{emergencyId}", produces = MediaType.APPLICATION_JSON_VALUE)
+    @IsUserSpecific(idToMatch = {"ownerId"}, bypassRoles = {Roles.ADMIN})
+    @GetMapping(value = "/emergency/owners/{ownerId}", produces = MediaType.TEXT_EVENT_STREAM_VALUE)
+    public Flux<EmergencyResponseDTO> getEmergencyVisitsByOwnerId(final @PathVariable String ownerId) {
+        return bffApiGatewayController.getEmergencyVisitsByOwnerId(ownerId);
+    }
+
+    @PostMapping(value = "/emergency")
```

Comment:
This part of your code could help me to create my endpoint for the reviews. Hopefully it will work, thanks.